packages/nextjs/app/_components/CompletedGrants.tsx

```diff
@@ -32,8 +32,10 @@ const CompletedGrantCard = ({ title, description, askAmount, builder, link, comp
     </div>
   );
 };
-export const CompletedGrants = async () => {
+
+export const CompletedGrants = async ({ view = "full" }) => {
```

Comment:
We could have used `boolean` since it only has two values,  maybe something like `limit = false`

Reply:
Thanks Shiv! The code feels much cleaner with a boolean. Used `reducedView`, if you see better `limit` feel free to change or lmk to change 🙌♥